### DISCUSSION TITLE
UI load and integration with Playwright

### DISCUSSION THREAD

**Discussion by Romarionijim** [locustio/locust#2798] [General]
Hello, I'm an automation engineer, and I am a but new to load testing, I'm researching automation load testing tools to implement in my current workplace, and I have a few small questions regarding Locust.

1. I saw in the docs a small test example, but wanted to understand more, in order to use Locust for load testing, can I utilize an already existing framework written in playwright? meaning if I have existing UI page object model framework written in Python and playwright, can I reuse them for load tests? 

2. does locust support load test for UI? because in the example it seems it support API endpoints and I think I'm missing the UI load testing to simulate thousands of users doing the same action in the UI.

Thank you in advance!

**Comment by cyberw** [comment]
Duplicate, right?
